docs/src/pages/api.md

```diff
@@ -97,6 +97,7 @@ run!{T<:DispatchNode, S<:DispatchNode}(exec::Executor, nodes::AbstractArray{T},
 run!(::Executor, ::DispatchGraph)
 prepare!(::Executor, ::DispatchGraph)
 dispatch!(::Executor, ::DispatchGraph)
+Dispatcher.run_inner_node!(::Executor, ::DispatchNode, ::Int)
```

Comment:
If this is in the API docs, the docstring should include a note saying that typical Executor implementations should not need to override it.